### PR TITLE
PLNSRVCE-794: Add verify-built-artifacts command

### PR DESCRIPTION
**Pull request by dwalluck**

### CODE REVIEW

File: java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/verifier/VerifyBuiltArtifactsCommand.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        var manager = system.newLocalRepositoryManager(session, localRepository);
+        session.setLocalRepositoryManager(manager);
+        var centralRepository = new Builder(DEFAULT_REMOTE_REPO_ID, "default", DEFAULT_REMOTE_REPO_URL).build();
+        var mirror = mirrorSelector.getMirror(centralRepository);
+
+        if (mirror != null) {
+            System.out.printf("Mirror for %s: %s%n", centralRepository.getId(), mirror);
+            remoteRepositories = List.of(mirror);
+        } else {
+            System.out.printf("Using repository URL %s%n", repositoryUrl);
```
Comment (stuartwdouglas) [inline java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/verifier/VerifyBuiltArtifactsCommand.java:76]: You can't do this in the constructor, the fields will not have been set yet.

Comment (dwalluck) [reply java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/verifier/VerifyBuiltArtifactsCommand.java]: Fixed it. Thanks.

**Pull request merged**